docs/README.md

```diff
@@ -13,9 +13,19 @@ This is the documentation of AMS, it's focused on the **0.10.x version.**
 ## How to
 
 - [How to add root key](howto/add_root_key.md)
-- [How to add pagination links](howto/add_pagination_links.md)
 - [Using AMS Outside Of Controllers](howto/outside_controller_use.md)
 - [How to use JSON API with Ember](howto/ember-and-json-api.md)
+- [How to add pagination links](howto/add_pagination_links.md)
+
+#### JSON API Resource `type`
+
+##### Implicit
+
+Resource class name is used, e.g. the type of a `Post` will be `posts` in the resulting JSON API resource. For namespaced and multiword classes, AMS will use underscores to seperate words and namespaces. e.g.: `Namespaced::MultiWordClass` becomes `namespaced_multi_word_class`.
+
```

Comment:
right now it uses `::` -> `/` but per discussion in jSON Api land I think it should be `__` since `/` is illegal per JSON API